### PR TITLE
BHBC-969

### PR DESCRIPTION
**Pull request by cgarrettjones**
<details open>
  <summary><a href="https://quartech.atlassian.net/browse/BHBC-969" title="BHBC-969" target="_blank">BHBC-969</a></summary>
  <br />
  <table>
    <tr>
      <th>Summary</th>
      <td>DB migration roles and system constants </td>
    </tr>
    <tr>
      <th>Type</th>
      <td>
        Story
        Story
      </td>
    </tr>
    <tr>
      <th>Status</th>
      <td>In Progress</td>
    </tr>
    <tr>
      <th>Points</th>
      <td>N/A</td>
    </tr>
    <tr>
      <th>Labels</th>
      <td>-</td>
    </tr>
  </table>
</details>

---

# Overview

## Links to jira tickets

- BHBC-969

## This PR contains the following changes

- added columns project_attachment.key, project_attachment.file_size
- populated system_roles and project_roles values

## This PR contains the following types of changes

- [x] New feature (change which adds functionality)
- [x] Enhancement (improvements to existing functionality)
- [ ] Bug fix (change which fixes an issue)
- [ ] Misc cleanup / Refactoring / Documentation
- [ ] Version change

## Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### General

- [x] I have performed a self-review of my own code

### Code

- [x] New files/classes/functions have appropriately descriptive names and comment blocks to describe their use/behaviour
- [x] I have avoided duplicating code when possible, moving re-usable pieces into functions
- [x] I have avoided hard-coding values where possible and moved any re-usable constants to a constants file
- [x] My code is as flat as possible (avoids deeply nested if/else blocks, promise chains, etc)
- [x] My code changes account for null/undefined values and handle errors appropriately
- [x] My code uses types/interfaces to help describe values/parameters/etc, help ensure type safety, and improve readability

### Style

- [x] My code follows the established style conventions
- [ ] My code uses native material-ui components/icons/conventions when possible

### Documentation

- [x] I have commented my code sufficiently, such that an unfamiliar developer could understand my code
- [ ] I have added/updated README's and related documentation, as needed

### Tests

- [ ] I have added/updated unit tests for any code I've added/updated
- [ ] I have added/updated the Postman requests/tests to account for any API endpoints I've added/updated

### Linting/Formatting

- [ ] I have run the linter and fixed any issues, as needed  
       _See the `lint` commands in package.json_
- [ ] I have run the formatter and fixed any issues, as needed  
       _See the `format` commands in package.json_

### SonarCloud

- [ ] I have addressed all SonarCloud Bugs, Vulnerabilities, Security Hotspots, and Code Smells

## How Has This Been Tested?

Smoke test.

## Screenshots

Please add any relevant UI screenshots, if applicable.

### CODE REVIEW

**Review by sdevalapurkar — DISMISSED** [review]
looks great!
just a few missing newlines at end of files, but not a real issue

File: database/src/migrations/release.0.10/smoketest_release.sql [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  delete from system_user where user_identifier = 'myIDIR';
+
+  insert into system_user (uis_id, user_identifier, record_effective_date, create_date, create_user) select id, 'myIDIR', now(), now(), 1 from user_identity_source where name = 'IDIR' and record_end_date is null;
+
+  select count(1) into __count from system_user;
+<<<<<<< HEAD
+  assert __count > 1, 'FAIL system_user';
+=======
+  assert __count = 2, 'FAIL system_user';
+>>>>>>> a39d7d62c5023a2196dac997b6878e6f1dd4e5b0
```
Comment (NickPhura) [inline database/src/migrations/release.0.10/smoketest_release.sql]: Looks like some leftover merge artifacts `>>>>` etc

File: database/src/migrations/release.0.10/db_setup_up.sql [hunk 2]
```diff
@@ -1,4 +1,6 @@
 -- db_setup_up.sql
+\c biohub
+
```
Comment (NickPhura) [inline database/src/migrations/release.0.10/db_setup_up.sql]: Is this needed? Looks like we didn't have it in past, maybe just something from when you were testing it?

File: database/src/migrations/release.0.10/smoketest_release.sql (hunk 1 above)
Comment (cgarrettjones) [reply database/src/migrations/release.0.10/smoketest_release.sql]: doesn't matter, just a piece of of smoke test code

File: database/src/migrations/release.0.10/db_setup_up.sql (hunk 2 above)
Comment (cgarrettjones) [reply database/src/migrations/release.0.10/db_setup_up.sql]: this file is not necessary and can be pulled.

**Review by sdevalapurkar — DISMISSED** [review]

**Review by sdevalapurkar — APPROVED** [review]

**Pull request merged**